pr-data.csv

```diff
@@ -1882,6 +1882,7 @@ https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.TaggedUnionsTest.testTaggedUnionsDisabled,ID,Opened,https://github.com/vojtechhabarta/typescript-generator/pull/688,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.TaggedUnionsTest.testTaggedUnionsWithExistingProperty,ID,Opened,https://github.com/vojtechhabarta/typescript-generator/pull/688,
 https://github.com/vojtechhabarta/typescript-generator,caef58bfe55a8cd1fc5d7368c19e9ff818a7b7fa,typescript-generator-core,cz.habarta.typescript.generator.TaggedUnionsTest.testTaggedUnionsWithInterfaces,ID,Opened,https://github.com/vojtechhabarta/typescript-generator/pull/688,
+https://github.com/webx/citrus,adfceaab120661e78ee7608f3335eaf863693a38,citrus/common/util/src/test/java/com/alibaba/citrus/util,com.alibaba.citrus.util.ClassUtilTests.java,NOD,,,
```

Comment:
The module path you provided looks abnormal. Did you mean to use `common/util` instead for the module path? The test name (`java`) also does not seem to exist in `ClassUtilTests`.

Reply:
I will look into it asap